addons/test_mail/tests/test_mail_alias_mixin.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.tests.common import tagged, users
+
+
+@tagged('mail_alias_mixin')
+class TestMailAliasMixin(MailCommon):
+
+    @users('admin')
+    def test_mc_alias_create(self):
+        """ Test the batch creation of container records like projects or teams in a multi-company
+        setup."""
+
+        vals = [{'company_id': company.id} for company in self.env['res.company'].search([])]
+        self.company_2.alias_domain_id = None
+        self.env['mail.test.container.mc'].create(vals)
```

Comment:
There's already `test_alias_mixin_alias_email` which seems like it should fit the bill, just updating that is fine I think

Would also avoid relying on the state of the DB, and thus demo data. You can just update one of the companies to have an empty alias from the start, or add a company that does. Then check that there indeed isn't an alias on the record created from that company.

Reply:
Thanks, will look into it and modify accordingly :saluting_face: